crates/db_views/src/vote_view.rs

```diff
@@ -59,7 +59,7 @@ impl VoteView {
 
     comment_like::table
       .inner_join(person::table)
-      .inner_join(post::table)
+      .inner_join(comment::table.inner_join(post::table))
```

Comment:
Is there a difference in logic or performance if we put each of these individually on the top level chain? i.e.

```rust
comment_like::table
    .inner_join(person::table)
    .inner_join(comment::table)
    .inner_join(post::table)
    // etc.
```

Reply:
There should be no difference because only 5 tables are joined, and the join collapse limit is set to 11, which means the database ignores the given join order